build-system/amp.extern.js

```diff
@@ -893,3 +901,14 @@ class FeaturePolicy {
  * @type {?FeaturePolicy}
  */
 HTMLIFrameElement.prototype.featurePolicy;
+
+
+/**
+ * @typedef {{
+ *   ISOCountry: string,
+ *   matchedISOCountryGroups: !Array<string>,
+ *   allISOCountryGroups: !Array<string>,
+ *   isInCountryGroup: (function(string): number),
+ * }}
+ */
+let GeoDef;
```

Comment:
GeoDef is exposed to amp-analytics as var substitutions so we do need to preserve the keys. i could also enforce this as a JsonObject but i thought this would better preserve the shape documentation wise in the extern